docs/src/main/tut/docs/README.md

```diff
@@ -63,10 +63,29 @@ Learn more about [algebras](./core/algebras) in the extended documentation.
 
 Freestyle algebras can be combined into `@module` definitions which provide aggregation and unification over the parameterization of Free programs.
 
+## Building programs
+
+Abstract definitions are all it takes to start building programs that support sequential and parallel 
+operations that are entirely decoupled from their runtime interpretation.
+
+The example below combines both algebras to produce a more complex program:
+
 ```tut:book
 @module trait Application {
   val validation: Validation
   val interaction: Interaction
+  
+  import cats.implicits._
+  
+  def program: FS.Seq[Unit] = 
+    for {
+      userInput <- interaction.ask("Give me something with at least 3 chars and a number on it")
+      valid     <- (validation.minSize(userInput, 3) |@| validation.hasNumber(userInput)).map(_ && _).freeS
```

Comment:
We should replace the cartesian builder syntax (the `|@|`) here (like you did in the _Combining `@tagless` and `@free` algebras_ section).

```scala
(validation.minSize(userInput, 3), validation.hasNumber(userInput)).mapN(_ && _).freeS
```

We could also do (but I think the for comprehension is better in this documentation) : 

```scala
interaction.ask("Give me something with at least 3 chars and a number on it").flatMap { userInput =>
  (validation.minSize(userInput, 3), validation.hasNumber(userInput))
    .mapN(_ && _)
    .freeS
    .ifM(
      interaction.tell("awesomesauce!"),
      interaction.tell(s"$userInput is not valid")
    )
}
```